zerver/lib/test_fixtures.py

```diff
@@ -21,6 +21,33 @@
 UUID_VAR_DIR = get_dev_uuid_var_path()
 FILENAME_SPLITTER = re.compile('[\W\-_]')
 
+def run_db_migrations(**options: Any) -> None:
+    for app_config in apps.get_app_configs():
+        if module_has_submodule(app_config.module, "management"):
+            import_module('.management', app_config.name)
+
+    app_label = options['app_label'] if options.get('app_label') else None
+    db = options.get('database', DEFAULT_DB_ALIAS)
+    migration_status_file = options.get('migration_status_file', 'migration_status_test')
+    out = StringIO()
+    call_command(
+        'migrate',
+        '--noinput',
+        app_label=app_label,
+        database=db,
+        settings=options.get('settings', os.environ['DJANGO_SETTINGS_MODULE']),
+        traceback=options.get('traceback', True),
+    )
+    call_command(
+        'get_migration_status',
+        '--output=%s' % (migration_status_file),
+        app_label=app_label,
+        database=db,
+        settings=options.get('settings', os.environ['DJANGO_SETTINGS_MODULE']),
+        traceback=options.get('traceback', True),
+    )
```

Comment:
Why do we need to check migration status?

Reply:
Basically this command checks the migration status but when the output param is specified it updates the `migration_status_test` or the `migration_status_dev` file so that the status is updated for the next round.

Reply:
OK